src/gwt/src/org/rstudio/core/client/command/ShortcutManager.java

```diff
@@ -718,6 +720,18 @@ public final boolean isIgnoredKeyCombination(KeyCombination keys)
       return ignoredKeys_.isIgnoredKeyCombination(keys);
    }
    
+   public void setReportShortcutBinding(boolean report)
+   {
+      reportShortcutBinding_ = report;
+   }
+
+   private boolean reportShortcutBinding(String message)
+   {
+      if (reportShortcutBinding_)
```

Comment:
When unchecked, checkable menu items look exactly like regular menu items so I don't think most people will know the first time that they invoke this mode that they need to go back to the menu to turn it off. 

I think that this should either be (a) self-cancelling (it just reports one shortcut and quits) or advertise a way to quit in the message itself (a special shortcut or a reference to the menu item).

Reply:
Good point, switched to a regular menu item, and rely on closing the MessageBar to turn off this mode (and mention that in the initial MessageBar text shown when activating this mode).